src/styles/style.css

```diff
@@ -0,0 +1,94 @@
+@import './libs/fonts.css';
+
+/* #region global */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  font-family: inherit;
+  font-weight: inherit;
+}
+
+/* #endregion */
+
+.search-bar {
```

Comment:
it will be nice practice to group your styles

example:

```
.search-bar {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    gap: 20px;
    width: 100%;
    margin-top: 20px;
}
```

but it's not necessary))

Reply:
noted)